app/service/rest.py

```diff
@@ -1130,8 +1131,8 @@ def purge_services_created_by(user_id):
         services = dao_fetch_all_services_created_by_user(user_id=user_id)
         for service in services:
             delete_service_and_all_associated_db_objects(service=service)
+        delete_invitations_sent_by_user(user_id=user_id)
     except Exception as e:
         return jsonify(result="error", message=f"Unable to purge services created by user {user_id}: {e}"), 500
 
-    service_list = ",".join([x["id"] for x in services])
-    return jsonify({"message": f"Successfully purged services: {service_list}"}), 200
+    return jsonify({"message": "Successfully purged services"}), 200
```

Comment:
I don't think there's anything wrong with returning this, but why was the specific service list removed from the response?

Reply:
I wasn't surfacing that data in the caller, and I'd need to type-check the services list, check for empty list, null response etc. It was simpler just to have a generic return message.